File: app/src/main/java/ru/MeatGames/roguelike/tomb/TileDB.java
package ru.MeatGames.roguelike.tomb;

import android.graphics.Bitmap;

public class TileDB {

	public boolean psb;
	public boolean vis;
	public boolean use;
    public boolean isWall;
	public Bitmap img;

    public TileDB(){
        isWall = false;
    }

}

File: app/src/main/java/ru/MeatGames/roguelike/tomb/TileDB.kt
package ru.MeatGames.roguelike.tomb

import android.graphics.Bitmap

class TileDB {

    var psb: Boolean = false
    var vis: Boolean = false
    var use: Boolean = false
    var isWall: Boolean = false
    var img: Bitmap? = null

    init {
        isWall = false
    }

}